src/reducers.js

```diff
@@ -103,7 +123,8 @@ function auth(state = { firebaseUser: null, isLoaded: false }, action) {
 const rootReducer = combineReducers({
   members,
   uidToOpMeta,
-  auth
+  auth,
+  operations,
```

Comment:
How is operations different from `uidToOpMeta`, do we need both?

Reply:
I can't evaluate that since i'm not familiar with firebase, etc. i kinda wish all these reducers weren't so tied up with hyper specific firebase vocabulary since it makes a newcomer like me have no idea what this is doing